.editorconfig

```diff
@@ -0,0 +1,28 @@
+root = true
+
+[*]
+charset = utf-8
+indent_size = 2
```

Comment:
Can we use 4 spaces, as suggested by the official convention - https://kotlinlang.org/docs/coding-conventions.html#indentation ?

Reply:
Yes! but for the compose project, I might prefer 2 spaces for indentation, as it looks more readable in cases with deep component nesting. However, if 4 spaces are to be used, that's fine too

Reply:
Got it. 
One more argument for keeping it 4 is that the repository is in Kotlin group, so using the official convention is preferred :)